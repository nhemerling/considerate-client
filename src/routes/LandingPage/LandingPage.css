.LandingPage__section h1 {
  font-size: 40px;
  text-align: center;
}

ul {
  list-style-type: none;
  margin: 0 10px;
  padding: 0;
}

@media screen and (min-width: 450px) {
  .LandingPage__section h1 {
    font-size: 50px;
  }
}
