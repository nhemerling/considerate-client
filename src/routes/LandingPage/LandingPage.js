import React, { Component } from 'react';
import './LandingPage.css';

export default class LandingPage extends Component {
  render() {
    return (
      <div className="LandingPage">
        <section className="LandingPage__section">
          <h1>How did you remember?!</h1>
        </section>

        <section className="LandingPage__section">
          <header>
            <h3>Demo Considerate</h3>
          </header>
          <p>If you'd like to demo Considerate, log in with:</p>
          <ul>
            <li>
              username: <b>demo</b>
            </li>
            <li>
              password: <b>D33mm0!!</b>
            </li>
          </ul>
        </section>

        <br />

        <section className="LandingPage__section">
          <header>
            <h3>Give the perfect gift, every time.</h3>
          </header>
          <p>
            Considerate helps you keep track of the things your friends like and
            of the occasions when you'd like to give them the perfect gift.
          </p>
          <p>You can be the thoughtful friend you've always meant to be.</p>
        </section>

        <br />

        <section className="LandingPage__section">
          <header>
            <h3>Keep track of what your friends like.</h3>
          </header>
          <p>
            The key to giving the perfect gift is remembering what your friends
            like.
          </p>
          <p>
            Considerate provides you with an intuitive interface that helps you
            keep track of all the things your friends have told you they like.
          </p>
        </section>

        <br />

        <section className="LandingPage__section">
          <header>
            <h3>We'll remind you.</h3>
          </header>
          <p>
            When you sign in, the home page will show you your friends with
            upcoming special occassions.
          </p>
          <p>That way you'll never miss another anniversay.</p>
        </section>

        <br />
      </div>
    );
  }
}
