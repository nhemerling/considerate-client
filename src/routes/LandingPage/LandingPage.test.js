import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from './LandingPage';

describe('LandingPage component', () => {
  it('renders a LandingPage component', () => {
    const wrapper = shallow(<LandingPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
