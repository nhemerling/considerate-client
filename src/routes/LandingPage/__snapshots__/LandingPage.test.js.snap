// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders a LandingPage component 1`] = `
<div
  className="LandingPage"
>
  <section
    className="LandingPage__section"
  >
    <h1>
      How did you remember?!
    </h1>
  </section>
  <section
    className="LandingPage__section"
  >
    <header>
      <h3>
        Give the perfect gift, every time.
      </h3>
    </header>
    <p>
      Considerate helps you keep track of the things your friends like and of the occasions when you'd like to give them the perfect gift.
    </p>
    <p>
      You can be the thoughtful friend you've always meant to be.
    </p>
  </section>
  <section
    className="LandingPage__section"
  >
    <header>
      <h3>
        Keep track of what your friends like.
      </h3>
    </header>
    <p>
      The key to giving the perfect gift is remembering what your friends like.
    </p>
    <p>
      Considerate provides you with an intuitive interface that helps you keep track of all the things your friends have told you they like.
    </p>
  </section>
  <section
    className="LandingPage__section"
  >
    <header>
      <h3>
        We'll remind you.
      </h3>
    </header>
    <p>
      When you sign in, the home page will show you your friends with upcoming special occassions.
    </p>
    <p>
      That way you'll never miss another anniversay.
    </p>
  </section>
  <section
    className="LandingPage__section"
  >
    <header>
      <h3>
        Demo Considerate
      </h3>
    </header>
    <p>
      If you'd like to demo Considerate, log in with the username 'demo' and the password 'D33mm0!'.
    </p>
  </section>
</div>
`;
